### PR TITLE
blueprint: Don't use replicate

### PR DESCRIPTION
**Pull request by kayousterhout**
We're planning to remove the replicate method from Quilt, so this
commit switches to using iteration to create the containers.

### CODE REVIEW

File: elasticsearch.js [hunk 1]
```diff
@@ -5,13 +5,13 @@ function getHostname(c) {
...
 
 function Elasticsearch(n) {
-  const ref = new quilt.Container('elasticsearch', 'elasticsearch:2.4', {
-    command: [
-      '--transport.tcp.port', `${this.transportPorts.min}-${this.transportPorts.max}`,
-      '--http.port', this.port.toString(),
-    ],
-  });
-  this.containers = ref.replicate(n);
+  this.containers = Array(n).fill().map(() =>
```
Comment (kayousterhout) [inline elasticsearch.js]: @kklin I realized the `.fill()` is necessary here (see https://stackoverflow.com/questions/12503146/create-an-array-with-same-element-repeated-multiple-times-in-javascript) -- curious to hear whether this changes your opinion about making this change (for now, I didn't do it in any of the other repos).

**Comment by kklin** [comment]
I'm going to hold off on reviewing this until after our meeting on Tuesday

**Comment by kayousterhout** [comment]
@kklin updated based on our conversation today

**Comment by kklin** [comment]
@kayousterhout linting failed

**Comment by kayousterhout** [comment]
Gah sorry fixed

**Pull request merged**